examples/c/nonlinear_chain_ocp_nlp.c

```diff
@@ -1316,17 +1328,18 @@ int main()
 
 				// Cyt
 				blasfeo_dgese(nu[i]+nx[i], ny[i], 0.0, &stage_cost_ls->Cyt, 0, 0);
-					for (int j = 0; j < nu[i]; j++)
-				BLASFEO_DMATEL(&stage_cost_ls->Cyt, j, nx[i]+j) = 1.0;
-					for (int j = 0; j < nx[i]; j++)
-				BLASFEO_DMATEL(&stage_cost_ls->Cyt, nu[i]+j, j) = 1.0;
+
+				for (int j = 0; j < nu[i]; j++)
+					BLASFEO_DMATEL(&stage_cost_ls->Cyt, j, nx[i]+j) = 1.0;
```

Comment:
this stuff still needs some setter

Reply:
it has a setter, taking a column-major matrix.
I was too lazy to understand all unpacks into the memory, create an equivalent double array and use the setter in every example.
The one example, in which I checked that only setters are used is the `wind_turbine_nmpc`.
I can change this to use the setter too, but doing it for all examples is a bit of work.